### PR TITLE
- Added persian translation

### PR DESCRIPTION
**Pull request by MatinHQ**
Persian translate which used Latin keywords AKA [Fingilish](https://en.wikipedia.org/wiki/Romanization_of_Persian#Fingilish)

You guys can convert Latin letters to native Persian with https://behnevis.com/en/ and use google translate to verify the translates

### CODE REVIEW

File: locales/pr.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	"refuel_cannot_afford": "Shoma pole kafi baraye benzin zadan nadarid",
+	"vehicle_far": "Vasile naghliye shoma dor ast",
+	"pump_fuel_with_can": "Baraye estefade az pomo benzin bayad dabe ro kenar bezarid",
+	"fuel_help": "Dokme ~INPUT_C2939D45~ jahat benzdin zadan",
+	"petrolcan_help": "Dokme ~INPUT_C2939D45~ jahat kharid ya por kardan dabe benzin",
+	"leave_vehicle": "Baraye benzin zadan bayad az vasile naghliye kharej shid",
+	"start_fueling": "Shoroe benzin zadan",
+	"petrolcan_buy_or_refill": "Buy or refill a fuel can",
+	"fuel_station_blip": "Pomp Benzin",
+	"not_enough_money": "Not enough money! Missing $%s",
```
Comment (LukeWasTakenn) [inline locales/pr.json]: This one is not translated

File: locales/pr.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	"tank_full": "Makhzan in vasile naghliye por ast",
+	"petrolcan_cannot_afford": "Shoma pole kafi baraye kharid dabe benzin nadarid",
+	"refuel_cannot_afford": "Shoma pole kafi baraye benzin zadan nadarid",
+	"vehicle_far": "Vasile naghliye shoma dor ast",
+	"pump_fuel_with_can": "Baraye estefade az pomo benzin bayad dabe ro kenar bezarid",
+	"fuel_help": "Dokme ~INPUT_C2939D45~ jahat benzdin zadan",
+	"petrolcan_help": "Dokme ~INPUT_C2939D45~ jahat kharid ya por kardan dabe benzin",
+	"leave_vehicle": "Baraye benzin zadan bayad az vasile naghliye kharej shid",
+	"start_fueling": "Shoroe benzin zadan",
+	"petrolcan_buy_or_refill": "Buy or refill a fuel can",
```
Comment (LukeWasTakenn) [inline locales/pr.json]: As well as this one

**Review by LukeWasTakenn — APPROVED** [review]

**Comment by LukeWasTakenn** [comment]
Meant to request changes but marked as approved : (

**Comment by MatinHQ** [comment]
> Meant to request changes but marked as approved : (

I have no idea how I missed those two, but corrected

**Pull request merged**